### PR TITLE
FIX: fix mutect2 meta id issue in tumor_only workflow

### PR DESCRIPTION
**Pull request by maxulysse**

### CODE REVIEW

File: subworkflows/local/bam_variant_calling_tumor_only_all/main.nf [hunk 1]
```diff
@@ -115,7 +115,7 @@ workflow BAM_VARIANT_CALLING_TUMOR_ONLY_ALL {
             cram.map{ meta, cram, crai ->
                 joint_mutect2 ?
                 [ meta + [ id:meta.patient ] - meta.subMap('sample', 'status', 'num_intervals', 'data_type', 'patient') , cram, crai ] :
-                [ meta - meta.subMap('sample', 'status', 'num_intervals', 'data_type'), cram, crai ]
+                [ meta + [ id:meta.sample ]- meta.subMap('sample', 'status', 'num_intervals', 'data_type'), cram, crai ]
```
Comment (FriederikeHanssen) [inline subworkflows/local/bam_variant_calling_tumor_only_all/main.nf:118]:
```suggestion
                [ meta + [ id:meta.sample ] - meta.subMap('sample', 'status', 'num_intervals', 'data_type'), cram, crai ]
```

Slightly confused why num_intervals a) is in there in hthe first place b) why it is removed. Don't you need it later to do proper merging across intervals or are you adding it back in then?

**Review by FriederikeHanssen — APPROVED** [review]

**Comment by maxulysse** [comment]
Taking forever on my tiny computer, so merging. Can you try out on your side, your machine is faster

**Comment by asp8200** [comment]
It also takes a loong time on my machine, but I'll kick it off now on the dev-branch

**Comment by maxulysse** [comment]
I thought you were doing all that on a bigger machine, not your own, to be honest.
Otherwise I would have continue on my side

**Comment by asp8200** [comment]
Well, I'm doing it on an aws-ec2. It has more muscles than my laptop

**Comment by FriederikeHanssen** [comment]
> Taking forever on my tiny computer, so merging. Can you try out on your side, your machine is faster

I think soon you get 🤠  :D

**Pull request merged**